.travis.yml

```diff
@@ -77,6 +77,9 @@ jobs:
       os: osx
       osx_image: xcode10.1
       compiler: clang
+      install:
+      - brew install ccache
+      - export PATH="/usr/local/opt/ccache/libexec:$PATH"
```

Comment:
I don't think you need this line, brew should link it to `/usr/local/bin` which I assume is already in the `$PATH`